pyproject.toml

```diff
@@ -113,17 +113,17 @@ changelog_ignore_labels = ["internal", "ci", "red-knot", "testing"]
 
 changelog_sections.breaking = "Breaking changes"
 changelog_sections.preview = "Preview features"
+changelog_sections.bug = "Bug fixes"
```

Comment:
Do I understand this correctly that an issue labeled with both `bug` and `preview` would still be categorized as `preview` (which I think is the correct behavior)


Site note: Sort of funny that we rely on the ordering of elements in a TOML table -- which is not specified.

Reply:
Yes, that's correct.